test/test_pipeline/test_tabular_regression.py

```diff
@@ -279,23 +288,18 @@ def test_set_range_search_space_updates(self, fit_dictionary_tabular):
             assert 'fully_connected:units_layer' in e.args[0]
 
 
-@pytest.mark.parametrize("fit_dictionary_tabular_dummy", ["regression"], indirect=True)
+@pytest.mark.parametrize("fit_dictionary_tabular_dummy", ['regression'], indirect=True)
 def test_pipeline_score(fit_dictionary_tabular_dummy):
     """This test makes sure that the pipeline is able to achieve a decent score on dummy data
     given the default configuration"""
+    # increase number of epochs to test for performance
+    fit_dictionary_tabular_dummy['epochs'] = 50
+
     X = fit_dictionary_tabular_dummy['X_train'].copy()
     y = fit_dictionary_tabular_dummy['y_train'].copy()
 
-    # lower the learning rate of the optimizer until seeding properly works
-    # with the default learning rate of 0.01 regression sometimes does not converge
     pipeline = TabularRegressionPipeline(
         dataset_properties=fit_dictionary_tabular_dummy['dataset_properties'],
-        search_space_updates=HyperparameterSearchSpaceUpdates([
-            HyperparameterSearchSpaceUpdate("optimizer",
-                                            "AdamOptimizer:lr",
-                                            value_range=[0.0001, 0.001],
-                                            default_value=0.001)
-        ])
```

Comment:
Could you explain why we can remove it? (I do not know what happens when we remove it)

Reply:
I think previously it was changing the default lr to 0.001 from 0.01 (which is the default in the AdamOptimizer class). As the comment says, it was to lower the learning rate until seeding worked properly. And as we have merged PR #172 we don't need it anymore. The pipeline converges with just 0.01 and hence we don't need to reduce the learning rate which was also slowing down the test (not significantly)